tests/keras_tests/feature_networks_tests/feature_networks/activation_16bit_test.py

```diff
@@ -98,7 +98,7 @@ def create_networks(self):
 
     def compare(self, quantized_model, float_model, input_x=None, quantization_info=None):
         mul1_act_quant = quantized_model.layers[3]
-        mul2_act_quant = quantized_model.layers[9]
+        mul2_act_quant = quantized_model.layers[10]
```

Comment:
we should use the util function get_layers_from_model_by_type if I'm not mistaken (should eliminate the need for doing such modification for tests when our inner representation is changing)